src/salaryAndBenefits/components/salaryCalculator/SalaryCalculator.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { useState } from "react";
+import styles from "./salaryCalculator.module.css";
+import InputField from "src/components/forms/inputField/InputField";
+import { RadioButtonGroup } from "src/components/forms/radioButtonGroup/RadioButtonGroup";
+import { calculateSalary } from "src/salaryAndBenefits/utils/calculateSalary";
+import Button from "src/components/buttons/Button";
+
+interface IOption {
+  id: string;
+  label: string;
+  disabled?: boolean;
+  currentChecked: boolean;
+}
+
+const options: IOption[] = [
+  { id: "bachelor", label: "Bachelor", currentChecked: false, disabled: false },
+  { id: "master", label: "Master", currentChecked: true, disabled: false },
+];
+
+interface SalaryCalculatorProps {
+    onSalaryUpdate: (salary: string) => void;
+  }
+
+export default function SalaryCalculator({ onSalaryUpdate }: SalaryCalculatorProps) {
+  const thisYear = new Date().getFullYear();
+  const [examinationYear, setExaminationYear] = useState(thisYear);
+  const [selectedDegree, setSelectedDegree] = useState("master"); 
```

Comment:
Side comment: Why do we default to master? I think we should have a chat with Even perhaps in terms of inclusivity when we default to a higher level education.

Reply:
Is it better to set it to bachelor maybe?

Reply:
[Legacy page](https://www.variant.no/kalkulator) defaults to bachelor 🤓